src/components/ui/googlePlacesSelect/index.tsx

```diff
@@ -44,6 +50,22 @@ export const GooglePlacesSelect = React.forwardRef<
       types: ['street_address', 'premise', 'postal_code', 'subpremise', 'route'],
     },
   })
+  // For some countries, the route type is required to show addresses options correctly
+  // But for US, if we allow users to use an address with the route type,
+  // Capitol Canary won't be able to successfully find the related congressperson based on the address provided
+  // That's why we need to filter out the route type for US addresses on required action flows
+  const usePlacesAutoCompleteResult = shouldLimitUSAddresses
+    ? data
+        ?.map(place =>
+          !place.description.includes('USA')
+            ? place
+            : !place.types.includes('route')
+              ? place
+              : null,
+        )
+        .filter(Boolean)
```

Comment:
Any reason not to do a single filter here?
```suggestion
    ? data?.filter(place => !place.description.includes('USA') || !place.types.includes('route'))
```

Reply:
Updated! Ty!